src/components/List.vue

```diff
@@ -74,6 +48,9 @@ export default defineComponent({
         console.log("couldn't load Exercises", e);
       }
     },
+    openExerciseDetail(exercise: Exercise) {
+      this.$router.push({ name: 'ExerciseDetail', params: { data: JSON.stringify(exercise) } });
```

Comment:
consider not stringifying the data, you should be able to pass it as object too

Reply:
done

Reply:
cause dosent work